src/FrameBuffer.cpp

```diff
@@ -176,20 +181,26 @@ void FrameBuffer::init(u32 _address, u16 _format, u16 _size, u16 _width, bool _c
 
 	const u16 maxHeight = VI_GetMaxBufferHeight(_width);
 	_initTexture(_width, maxHeight, _format, _size, m_pTexture);
+	_initTexture(_width, maxHeight, _format, _size, m_pFrameBufferCopyTexture);
```

Comment:
I don't like that copy texture created for every buffer, even if it will never use it.
Lets do lazy initialization of copy texture and copy FBO.  For example, FrameBuffer::_copyFrameBufferTexture() can check for copy texture and init if necessary.